src/pages/signIn/validInput.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import {PropTypes} from 'prop-types'
+
+import { Field } from 'formik'
+import { ValidContainer,StyledField } from './styled'
+
+const ValidInput=({type,error,name,label,autoComplete})=>{
```

Comment:
prettier formatting!
I will not ask you to do it again, just will not approve while it's not ready. I told you a lot about it, almost in all PRs